src/components/PopupWithForm.js

```diff
@@ -28,4 +35,12 @@ export default class PopupWithForm extends Popup {
       //this.close();
     });
   }
+
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._form
  }


```